bucket/cura.json

```diff
@@ -3,12 +3,9 @@
     "description": "Model editing tools for 3D printing",
     "homepage": "https://ultimaker.com/en/products/ultimaker-cura-software",
     "license": "LGPL-3.0-only",
-    "suggest": {
-        "Microsoft Visual C++ Redistributables 2015": "extras/vcredist2015"
-    },
```

Comment:
Why did you remove this suggest? Where is stated that now the application do not require vcredist?